### PR TITLE
Trim leading en trailing spaces

### PR DESCRIPTION
**Pull request by jaapmarcus**
https://forum.hestiacp.com/t/how-to-tweak-the-js-codes-in-the-admin-panel/13177/2

### CODE REVIEW

**Comment by AlecRust** [comment]
Hmm. Won't this mean if a user types a space e.g. before typing something else in an input, it will be trimmed?

**Comment by jaapmarcus** [comment]
You should be able to type in " domain.com" but it will be trimmed to `domain.com` 

I don't think there are any input fields where we expect a leading space

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/assets/9754650/6cef2367-d37b-4d00-8c8f-31e5f14212c6

File: web/js/src/trimInput.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//trim leading / trailing spaces form input fields
+export default function trimInput() {
```
Comment (AlecRust) [inline web/js/src/trimInput.js]: Maybe `handleAutoTrimInputs` would be more consistent?

**Review by AlecRust — APPROVED** [review]

File: web/js/src/trimInput.js (hunk 1 above)
Comment (jaapmarcus) [reply web/js/src/trimInput.js]: Fine for me ..

Comment (jaapmarcus) [reply web/js/src/trimInput.js]: Renamed function to handleAutoTrimInputs and filename to autoTrimInputs
...

**Pull request merged**